packages/bpk-docs/src/components/DocsPageBuilder/DocsPageBuilder.js

```diff
@@ -283,6 +288,16 @@ DocsPageBuilder.propTypes = {
           subText: PropTypes.string.isRequired,
         }),
       ),
+      videos: PropTypes.arrayOf(
+        PropTypes.shape({
+          title: PropTypes.string.isRequired,
+          src: PropTypes.string.isRequired,
+          width: PropTypes.number.isRequired,
+          height: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
```

Comment:
I'm surprised this works, the spec says you can't use strings/percentages for height, it must be a number of pixels.